internal-services/catalog/create-advisory-pipeline.yaml

```diff
@@ -31,6 +31,9 @@ spec:
     - name: advisory_secret_name
       type: string
       description: The name of the secret that contains the advisory creation metadata
+    - name: errata_secret_name
```

Comment:
Would it make sense to set a default here? Perhaps the stage one? That way you don't have to worry about syncing the change in the catalog repo. But maybe it's ok if you make sure to change catalog first. Just a thought.

Reply:
If only this task had been migrated to the internal dir in the catalog repo!
I think it is better to be consistent and follow how advisory_secret_name works even if it means we have to merge both PRs around the same time. I personally don't want to add code that we are only adding because of a plumbing limitation